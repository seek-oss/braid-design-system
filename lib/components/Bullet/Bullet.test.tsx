import React from 'react';
import { Bullet } from '../';
import { render } from 'react-testing-library';
import forEachTheme from '../../test/utils/forEachTheme';

describe('Bullet', () => {
  forEachTheme(({ ThemeProvider }) => {
    test('Default', () => {
      const { container } = render(
        <ThemeProvider>
          <Bullet>Bullet</Bullet>
        </ThemeProvider>
      );
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
