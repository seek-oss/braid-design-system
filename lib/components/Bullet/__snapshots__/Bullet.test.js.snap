// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bullet Theme: jobStreet Default 1`] = `
<li
  class="Text__block Text__listItem Box__root borderColor__default paddingBottom__smaller"
  style="margin-left: 1.3em;"
>
  <span
    class="Text__block color__neutral fontFamily__text fontSize__standard fontWeight__regular"
  >
    Bullet
  </span>
</li>
`;

exports[`Bullet Theme: seekAnz Default 1`] = `
<li
  class="Text__block Text__listItem Box__root borderColor__default paddingBottom__smaller"
  style="margin-left: 1.3em;"
>
  <span
    class="Text__block color__neutral fontFamily__text fontSize__standard fontWeight__regular"
  >
    Bullet
  </span>
</li>
`;

exports[`Bullet Theme: seekAsia Default 1`] = `
<li
  class="Text__block Text__listItem Box__root borderColor__default paddingBottom__smaller"
  style="margin-left: 1.3em;"
>
  <span
    class="Text__block color__neutral fontFamily__text fontSize__standard fontWeight__regular"
  >
    Bullet
  </span>
</li>
`;

exports[`Bullet Theme: wireframe Default 1`] = `
<li
  class="Text__block Text__listItem Box__root borderColor__default paddingBottom__smaller"
  style="margin-left: 1.3em;"
>
  <span
    class="Text__block color__neutral fontFamily__text fontSize__standard fontWeight__regular"
  >
    Bullet
  </span>
</li>
`;
